### PR TITLE
Task-J41 add episode validation

### PR DESCRIPTION
**Pull request by vitalik-prog**

### CODE REVIEW

File: backend/src/api/episodes/episodes.api.ts [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import { Router } from 'express';
+import {
+  addEpisode as addEpisodeValidationSchema,
```
Comment (ghost) [inline backend/src/api/episodes/episodes.api.ts]:
```suggestion
  episode as episodeValidationSchema,
```

File: backend/src/validation-schemas/validation-schemas.ts [hunk 2]
```diff
@@ -1 +1,2 @@
-export { signUp, signIn } from 'jabber-shared/validation-schemas/validation-schemas';
+export { signUp, signIn, addEpisode } from 'jabber-shared/validation-schemas/validation-schemas';
```
Comment (ghost) [inline backend/src/validation-schemas/validation-schemas.ts]:
```suggestion
export { signUp, signIn, episode } from 'jabber-shared/validation-schemas/validation-schemas';
```

File: shared/src/common/enums/enums.ts [hunk 3]
```diff
@@ -19,6 +19,8 @@ export {
 export {
   SignUpValidationRule,
   SignUpValidationMessage,
+  AddEpisodeValidationRule,
+  AddEpisodeValidationMessage,
```
Comment (ghost) [inline shared/src/common/enums/enums.ts]:
```suggestion
  EpisodeValidationRule,
  EpisodeValidationMessage,
```

File: shared/src/common/enums/validation/add-episode-validation-message.enum.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { AddEpisodeValidationRule } from './add-episode-validation-rule.enum';
+
+const AddEpisodeValidationMessage = {
```
Comment (ghost) [inline shared/src/common/enums/validation/add-episode-validation-message.enum.ts]:
```suggestion
const EpisodeValidationMessage = {
```

File: shared/src/common/enums/validation/add-episode-validation-message.enum.ts [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { AddEpisodeValidationRule } from './add-episode-validation-rule.enum';
```
Comment (ghost) [inline shared/src/common/enums/validation/add-episode-validation-message.enum.ts]: rename to `shared/src/common/enums/validation/episode-validation-message.enum.ts`

File: shared/src/common/enums/validation/add-episode-validation-rule.enum.ts [hunk 6]
```diff
@@ -0,0 +1,6 @@
+enum AddEpisodeValidationRule {
```
Comment (ghost) [inline shared/src/common/enums/validation/add-episode-validation-rule.enum.ts]: rename `shared/src/common/enums/validation/episode-validation-rule.enum.ts`
```suggestion
enum EpisodeValidationRule {
```

File: shared/src/common/enums/validation/validation.ts [hunk 7]
```diff
@@ -2,5 +2,7 @@ export { SignInValidationRule } from './sign-in-validation-rule.enum';
 export { SignInValidationMessage } from './sign-in-validation-message.enum';
 export { SignUpValidationMessage } from './sign-up-validation-message.enum';
 export { SignUpValidationRule } from './sign-up-validation-rule.enum';
+export { AddEpisodeValidationMessage } from './add-episode-validation-message.enum';
+export { AddEpisodeValidationRule } from './add-episode-validation-rule.enum';
 export { PodcastValidationMessage } from './podcast-validation-message.enum';
-export { PodcastValidationRule } from './podcast-validation-rule.enum';
+export { PodcastValidationRule } from './podcast-validation-rule.enum';
```
Comment (ghost) [inline shared/src/common/enums/validation/validation.ts]: ll is required

File: shared/src/validation-schemas/episode/add/add-episode.validation-schema.ts [hunk 8]
```diff
@@ -0,0 +1,35 @@
+import * as Joi from 'joi';
+import {
+  AddEpisodeValidationRule,
+  AddEpisodeValidationMessage,
+  EpisodeCreatePayloadKey,
+} from '~/common/enums/enums';
+import { EpisodeCreatePayload } from '~/common/types/types';
+
+const addEpisode = Joi.object<EpisodeCreatePayload>({
```
Comment (ghost) [inline shared/src/validation-schemas/episode/add/add-episode.validation-schema.ts]:
```suggestion
const episode = Joi.object<EpisodeCreatePayload>({
```

**Review by ghost — CHANGES_REQUESTED** [review]

File: backend/src/api/episodes/episodes.api.ts (hunk 1 above)
Comment (vitalik-prog) [reply backend/src/api/episodes/episodes.api.ts]: fixed it in next commit

File: backend/src/validation-schemas/validation-schemas.ts (hunk 2 above)
Comment (vitalik-prog) [reply backend/src/validation-schemas/validation-schemas.ts]: fixed it in next commit

File: shared/src/common/enums/enums.ts (hunk 3 above)
Comment (vitalik-prog) [reply shared/src/common/enums/enums.ts]: fixed it in next commit

File: shared/src/common/enums/validation/add-episode-validation-message.enum.ts (hunk 4 above)
Comment (vitalik-prog) [reply shared/src/common/enums/validation/add-episode-validation-message.enum.ts]: fixed it in next commit

File: shared/src/common/enums/validation/add-episode-validation-message.enum.ts (hunk 5 above)
Comment (vitalik-prog) [reply shared/src/common/enums/validation/add-episode-validation-message.enum.ts]: fixed it in next commit

File: shared/src/common/enums/validation/add-episode-validation-rule.enum.ts (hunk 6 above)
Comment (vitalik-prog) [reply shared/src/common/enums/validation/add-episode-validation-rule.enum.ts]: fixed it in next commit

File: shared/src/validation-schemas/episode/add/add-episode.validation-schema.ts (hunk 8 above)
Comment (vitalik-prog) [reply shared/src/validation-schemas/episode/add/add-episode.validation-schema.ts]: fixed it in next commit

File: shared/src/common/enums/validation/validation.ts (hunk 7 above)
Comment (vitalik-prog) [reply shared/src/common/enums/validation/validation.ts]: i don't understand. Pls what is ll?

**Review by ghost — APPROVED** [review]

**Pull request merged**